watertap/tools/analysis_tools/loop_tool/tests/test_expected_diff_directory.yaml

```diff
@@ -0,0 +1,64 @@
+ro_diff_analysis:
+  membrane_cost:
+    simulation_setup:
+      build_defaults: {}
+      diff_params:
+        fs.costing.reverse_osmosis.membrane_cost:
+          diff_mode: percentile
+          diff_sample_type: UniformSample
+          nominal_lb: 10
+          nominal_ub: 30
+          num_samples: 1
+          param: fs.costing.reverse_osmosis.membrane_cost
+          relative_lb: -0.01
+          relative_ub: -0.01
+      diff_samples: 1
+      dir: D:\OneDrive\NAWI_work\Analysis\WaterTAP\watertap-dev\watertap\tools\analysis_tools\loop_tool\tests/output/ro_with_erd/ro_diff_analysis/membrane_cost
```

Comment:
We should set up temporary paths for testing, ones that are compatible with both windows and unix.

Reply:
Those don't need to be tested I guess I should just remove them from the test files. The pathing get tested when loop tool runs.